Makefile

```diff
@@ -1,3 +1,4 @@
+export GOFLAGS = -mod=readonly
```

Comment:
@isobit is this something you think we should add? 

@stephengroat-dd any chance you could provide some details about what this problem is solving?

Reply:
This isn't necessary now that https://github.com/remind101/ssm-env/pull/42 is merged.

From https://golang.org/ref/mod:
> Since Go 1.16, the go command acts as if -mod=readonly were set instead: if any changes to go.mod are needed, the go command reports an error and suggests a fix.